src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -160,21 +161,34 @@
            ::show-transaction-error error_message}
           {:db (update-in db [:wallet :transactions-unsigned] dissoc id)})))))
 
+(defn prepare-unconfirmed-transaction [db now hash id]
+  (-> (get-in db [:wallet :transactions-unsigned id])
```

Comment:
I would prefer
```
(-> db
    (get-in [:wallet :transactions-unsigned id])
    (assoc ...
```

Reply:
but it is better to have only one kind of data in a threading macro, so the first line gets a transaction from the db and next lines are changing this transaction. If you separate the first line then the first line you have a db, then a transaction, then you change this transaction

Reply:
Didn't get your point, but it seems like style differences for me. Usually i start threading macro with data structure i'm working on, then apply transformation/extraction step-by-step. In your case, first line of threading macro is already an operation. Consider this example from 
https://github.com/bbatsov/clojure-style-guide
```
(-> [1 2 3]
    reverse
    (conj 4)
    prn)
```
or
```
(-> (reverse [1 2 3])
    (conj 4)
    prn)
```

Reply:
This example is slightly different, but the idea that first line of threading macro doesn't start with operation, but with data structure / object

Reply:
Ok I changed it in a way that should please both approaches.

http://tonsky.me/blog/readable-clojure/#dont-spare-names